README.md

```diff
@@ -1,10 +1,9 @@
 # phimpme-android
-<img src="/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png" align="left" width="200" hspace="10" vspace="10">
+![Phimpme](/docs/_static/logo.png)
```

Comment:
Honestly, I'm not sure if I'd be able to fix this issue. I'm trying a lot of things but didn't have any good results till now. Better not depend on it.